package.json

```diff
@@ -25,14 +25,12 @@
     "Siebrand Mazeland"
   ],
   "devDependencies": {
-    "eslint-config-wikimedia": "^0.7.2",
-    "eslint-plugin-qunit": "^3.3.0",
+    "eslint-config-wikimedia": "^0.9.0",
     "grunt": "^1.0.3",
     "grunt-contrib-qunit": "^3.0.1",
     "grunt-eslint": "21.0.0",
-    "grunt-stylelint": "0.10.0",
-    "stylelint": "9.2.0",
```

Comment:
Is it right to remove it? Even if it's a dependency for other packages, isn't it better to put it here explicitly?

Reply:
grunt-stylelint is explicitily calling the right version, so the advantage is that some dev in this specific repo cannot break it by just falsely amending the stylelint version here.